testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCheckConstraintCompatibility.java

```diff
@@ -199,6 +204,46 @@ public void testCheckConstraintMergeCompatibility()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, DELTA_LAKE_EXCLUDE_104, DELTA_LAKE_EXCLUDE_113, PROFILE_SPECIFIC_TESTS})
```

Comment:
This PT could be a delta lake oss only PT.
Do you see any eventual functionality collision potential with DBX?

Reply:
I knew it could be OSS only, but I believe there's no such consensus yet.